### PR TITLE
User model

### PR DESCRIPTION
**Pull request by yovanny-wolox**
## Summary

- user model created
- migrations added
- packages updated

## Known Issues

- none

## Trello Card

https://trello.com/c/K8ZHsv4p/21-modelo-de-usuario

## Screenshot
Screenshot from 2021-07-12 16-14-11

### CODE REVIEW

File: app/models/user.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    'user',
+    {
+      id: {
+        primaryKey: true,
+        type: DataTypes.INTEGER,
+        allowNull: false,
+        unique: true,
+        autoIncrement: true
+      },
+      first_name: {
```
Comment (marioportela1993) [inline app/models/user.js]: Set fields in models as camelCase

File: app/models/user.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        type: DataTypes.STRING,
+        allowNull: false
+      },
+      last_name: {
```
Comment (marioportela1993) [inline app/models/user.js]: Same comment

File: migrations/migrations/20210712195841-create-user.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        type: Sequelize.DATE,
+        defaultValue: Sequelize.NOW
+      },
+      updated_at: {
+        allowNull: false,
+        type: Sequelize.DATE,
+        defaultValue: Sequelize.NOW
+      }
+    }),
+  down: queryInterface => queryInterface.dropTable('Users')
```
Comment (marioportela1993) [inline migrations/migrations/20210712195841-create-user.js]: Fix the table name please, the first letter is in upper case

File: migrations/migrations/20210712195841-create-user.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+module.exports = {
+  up: (queryInterface, Sequelize) =>
+    queryInterface.createTable('users', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      first_name: {
```
Comment (marioportela1993) [inline migrations/migrations/20210712195841-create-user.js:12]: Here is ok in snake_case

File: package.json [hunk 5]
```diff
@@ -73,8 +74,8 @@
     "express-wolox-logger": "^2.0.0",
     "factory-girl": "^5.0.4",
     "jwt-simple": "^0.5.1",
-    "pg": "^7.4.1",
-    "sequelize": "^5.21.5",
+    "pg": "^8.6.0",
```
Comment (marioportela1993) [inline package.json:77]: Nice

**Review by marioportela1993 — CHANGES_REQUESTED** [review]
Excellent job, Just a few comments

File: app/models/user.js (hunk 1 above)
Comment (yovanny-wolox) [reply app/models/user.js]: I had interpreted it as snake_case. Thanks for the feedback

File: package.json (hunk 5 above)
Comment (yovanny-wolox) [reply package.json:77]: Thanks!

File: migrations/migrations/20210712195841-create-user.js (hunk 4 above)
Comment (yovanny-wolox) [reply migrations/migrations/20210712195841-create-user.js:12]: Thanks!

File: migrations/migrations/20210712195841-create-user.js (hunk 3 above)
Comment (yovanny-wolox) [reply migrations/migrations/20210712195841-create-user.js]: Oops!

File: app/models/user.js (hunk 2 above)
Comment (yovanny-wolox) [reply app/models/user.js]: Done!

**Review by marioportela1993 — APPROVED** [review]

**Pull request merged**